Submodule fetch tag (Assimp)

I just went across an issue with Assimp when configuring Radium with
```
cmake ../Radium-Engine/ -DCMAKE_BUILD_TYPE=Release -DRADIUM_IO_ASSIMP=ON -DRADIUM_SKIP_IO_EXTERNAL=OFF
```
I get the following issue that was not there before:
```
[100%] Built target assimp
-- [addExternalFolder] Build ended
-- [addExternalFolder] Configure package file /home/hoshiryu/dev/Radium/build_radium_release/src/IO/external/cmake/package-IO.cmake
CMake Error at Bundle-GNU/lib/cmake/assimp-5.0/assimpTargets.cmake:71 (message):
  The imported target "assimp::assimp" references the file     "/lib/libassimp.so.5"  but this file does not exist.  Possible reasons include:  * The file was deleted, renamed, or moved to another location.  * An install or uninstall procedure did not complete successfully.  * The installation package was faulty and contained     "/home/hoshiryu/dev/Radium/Radium-Engine/Bundle-GNU/lib/cmake/assimp-5.0/assimpTargets.cmake"  but not all the files it references.Call Stack (most recent call first):
  Bundle-GNU/lib/cmake/assimp-5.0/assimp-config.cmake:1 (include)
  src/IO/CMakeLists.txt:25 (find_package)
``` 
It actually seems to come from new commits to the AssImp repository's master branch.
When I change the AssImp fetched branch from `origin/master` to `origin/assimp_5.0_release` in file `src/IO/external/CMakeList.txt` all goes right.

--- a/src/Core/external/CMakeLists.txt
+++ b/src/Core/external/CMakeLists.txt
@@ -14,9 +14,9 @@ set_property(DIRECTORY PROPERTY EP_STEP_TARGETS install)
 set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 
 ExternalProject_Add(Eigen3
-    GIT_REPOSITORY https://github.com/eigenteam/eigen-git-mirror.git
-    GIT_TAG origin/master
-    GIT_SHALLOW TRUE
+    GIT_REPOSITORY https://gitlab.com/libeigen/eigen.git
+    GIT_TAG e80ec243
+    GIT_SHALLOW FALSE
     GIT_PROGRESS TRUE
     INSTALL_DIR "${CMAKE_INSTALL_PREFIX}"
     CMAKE_ARGS
@@ -33,7 +33,7 @@ ExternalProject_Add(Eigen3
 # TODO, find why only OpenMesh is problematic
 ExternalProject_Add(OpenMesh
     GIT_REPOSITORY https://www.graphics.rwth-aachen.de:9000/OpenMesh/OpenMesh.git
-    GIT_TAG origin/master
+    GIT_TAG tags/OpenMesh-8.1
     GIT_SHALLOW TRUE
     GIT_PROGRESS TRUE
     INSTALL_DIR "${CMAKE_INSTALL_PREFIX}"
@@ -46,7 +46,7 @@ ExternalProject_Add(OpenMesh
 
 ExternalProject_Add(cpplocate
     GIT_REPOSITORY https://github.com/cginternals/cpplocate.git
-    GIT_TAG origin/master
+    GIT_TAG tags/v2.2.0
     GIT_SHALLOW TRUE
     GIT_PROGRESS TRUE
     INSTALL_DIR "${CMAKE_INSTALL_PREFIX}"

--- a/src/IO/external/CMakeLists.txt
+++ b/src/IO/external/CMakeLists.txt
@@ -19,7 +19,7 @@ if (RADIUM_IO_ASSIMP)
     # For now, only configure in Debug mode to work with xcode
     ExternalProject_Add(assimp
             GIT_REPOSITORY https://github.com/assimp/assimp.git
-            GIT_TAG origin/master
+            GIT_TAG tags/v5.0.1
             GIT_SHALLOW TRUE
             GIT_PROGRESS TRUE
             INSTALL_DIR "${CMAKE_INSTALL_PREFIX}"
@@ -39,7 +39,7 @@ endif()
 if(RADIUM_IO_TINYPLY)
     ExternalProject_Add( tinyply
         GIT_REPOSITORY https://github.com/ddiakopoulos/tinyply.git
-        GIT_TAG origin/master
+        GIT_TAG tags/2.3.2
         GIT_SHALLOW TRUE
         GIT_PROGRESS TRUE
         INSTALL_DIR "${CMAKE_INSTALL_PREFIX}"

